x-pack/plugins/ingest_pipelines/public/index.ts

```diff
@@ -13,3 +13,5 @@ export function plugin() {
 
 export { INGEST_PIPELINES_APP_LOCATOR, INGEST_PIPELINES_PAGES } from './locator';
 export type { IngestPipelinesListParams } from './locator';
+export { apiService } from './application/services/api';
+export type { Pipeline } from '../common/types';
```

Comment:
Exposing the service like this is risky, and strongly discouraged.

Ideally, the APIs would be exposed as part of the plugin's `start` contract.

Reply:
Updated.